pkgs/by-name/op/opentelemetry-cpp/package.nix

```diff
@@ -85,11 +85,6 @@ stdenv.mkDerivation (finalAttrs: {
     "dev"
   ];
 
-  postInstall = ''
-    substituteInPlace $out/lib/cmake/opentelemetry-cpp/opentelemetry-cpp*-target.cmake \
-      --replace-quiet "\''${_IMPORT_PREFIX}/include" "$dev/include"
-  '';
-
```

Comment:
Why was this removed? 
This seems to break stuff that depends on `opentelemetry-cpp.dev`, as the INTERFACE_INCLUDE_DIRECTORIES will have a reference to `${_IMPORT_PREFIX}`, which points to $out, not $dev; so the path doesn't exist by the time it's used as a depencency.

(eg in `$dev/lib/cmake/opentelemetry-cpp/opentelemetry-cpp-exporters_otlp_grpc-target.cmake`:
```
set_target_properties(opentelemetry-cpp::otlp_grpc_client PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "opentelemetry-cpp::sdk;opentelemetry-cpp::common"
)
```